geonode/base/templates/base/thumbnail_upload.html

```diff
@@ -20,7 +20,7 @@ <h2>{% trans "Upload a Thumbnail" %}</h2>
     <!-- current uploaded thumb -->
     {% if resource.curatedthumbnail %}
       <h5>Current thumbnail</h5>
-      <img src='{{ resource.curatedthumbnail.thumbnail_url }}'></img>
+      <img src='{{ resource.thumbnail_url }}'></img>
```

Comment:
by removing the curated_thumbnail what happens if we upload a new thumb from the document page?